source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACROpenURLTarget.h

```diff
@@ -4,10 +4,11 @@
 //
 //  Copyright © 2017 Microsoft. All rights reserved.
 //
-
+#import <SafariServices/SafariServices.h>
```

Comment:
This import would be better in the .mm file. Callers of ACROpenURLTarget don't need to include SafariServices.

Reply:
Thanks for catching that!

---
In reply to: [164588528](https://github.com/Microsoft/AdaptiveCards/pull/1075#discussion_r164588528) [](ancestors = 164588528)